internal/dcache/rpc/models.thrift

```diff
@@ -64,6 +64,21 @@ struct PutChunkResponse {
     3: list<RVNameAndState> componentRV
 }
 
+struct PutChunkExRequest {
+    1: PutChunkRequest request,
+    2: list<RVNameAndState> nextRVs
```

Comment:
Since we don't want the forwarding nodes to have any intelligence wrt handling RV states, this should be just a list of RV names.

Reply:
Updated to have just RV name